2024-06-06 20:40:41 - CarbonTracker: The following components were found: GPU with device(s) NVIDIA GeForce RTX 4090 Laptop GPU.
2024-06-06 20:42:53 - CarbonTracker: Average carbon intensity during training was 374.46 gCO2/kWh at detected location: Santiago, Santiago Metropolitan, CL.
2024-06-06 20:42:53 - CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:02:11
	Energy:	0.004266969129 kWh
	CO2eq:	1.597823042251 g
	This is equivalent to:
	0.014863470160 km travelled by car
2024-06-06 20:42:53 - CarbonTracker: Live carbon intensity could not be fetched at detected location: Santiago, Santiago Metropolitan, CL. Defaulted to average carbon intensity for CL in 2021 of 374.46 gCO2/kWh. at detected location: Santiago, Santiago Metropolitan, CL.
2024-06-06 20:42:53 - CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:02:11
	Energy:	0.004266969129 kWh
	CO2eq:	1.597823042251 g
	This is equivalent to:
	0.014863470160 km travelled by car
2024-06-06 20:42:53 - CarbonTracker: Finished monitoring.
2024-06-06 20:44:57 - CarbonTracker: The following components were found: GPU with device(s) NVIDIA GeForce RTX 4090 Laptop GPU.
2024-06-06 20:47:20 - CarbonTracker: Traceback (most recent call last):
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 293, in epoch_end
    self.tracker.epoch_end()
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 154, in epoch_end
    self._log_epoch_measurements()
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 172, in _log_epoch_measurements
    power_avg = np.mean(comp.power_usages[-1], axis=0)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\fromnumeric.py", line 3504, in mean
    return _methods._mean(a, axis=axis, dtype=dtype,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\_methods.py", line 102, in _mean
    arr = asanyarray(a)
          ^^^^^^^^^^^^^
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (15,) + inhomogeneous part.

2024-06-06 20:48:04 - CarbonTracker: The following components were found: GPU with device(s) NVIDIA GeForce RTX 4090 Laptop GPU.
2024-06-06 20:50:16 - CarbonTracker: Traceback (most recent call last):
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 293, in epoch_end
    self.tracker.epoch_end()
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 154, in epoch_end
    self._log_epoch_measurements()
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\carbontracker\tracker.py", line 172, in _log_epoch_measurements
    power_avg = np.mean(comp.power_usages[-1], axis=0)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\fromnumeric.py", line 3504, in mean
    return _methods._mean(a, axis=axis, dtype=dtype,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\EVILAB\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\_methods.py", line 102, in _mean
    arr = asanyarray(a)
          ^^^^^^^^^^^^^
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (14,) + inhomogeneous part.

